### DISCUSSION TITLE
Is it possible to access metadata from yaml front matter in .qmd during execution?

### DISCUSSION THREAD

**Discussion by mfherman** [quarto-dev/quarto-cli#5753] [Q&A] [answered]
### Description

I know we can access `parameters` that are included as `params` in the yaml header as described here: https://quarto.org/docs/computations/parameters.html#knitr.

Is it also possible access and use other yaml fields in during execution/computation of a `.qmd` file? For instance is something like this possible?

````
---
title: "My Document"
author: "Matt Herman"
---

```{r}
yaml$author
```
````

**Comment by cscheid** [comment, marked as answer]
https://stackoverflow.com/questions/31220940/how-to-access-yaml-metadata-from-knitr

**Reply by mfherman** [reply]
Thank you! My googling is clearly subpar. I wonder if it's worth documenting somewhere on the Quarto site, since it seems to be a common question?

**Reply by mcanouil** [reply]
The issue is that it's 100 % `knitr`/`rmarkdown` and it's actually documented in a book: <https://bookdown.org/yihui/rmarkdown-cookbook/document-metadata.html>.

Also, as listed in https://github.com/quarto-dev/quarto-cli/discussions/5753#discussioncomment-6041773, a search in GitHub Issues/Discussions leads to the same results.

**Comment by mcanouil** [comment]
Related discussions:
- #4688
- #5159
- #3263
- #2121
- #3278
- #85

There are probably more.
